torch/_inductor/codecache.py

```diff
@@ -514,6 +515,17 @@ def __bool__(self):
 
             return True
 
+@dataclasses.dataclass
+class VecNEON(VecISA):
+    _bit_width = 256
```

Comment:
Neon's bit width is 128. Do you need to use 256 in order to get it to use 256 in `VecReduceAllSIMD`? If so, I recommend writing a comment about that here.

Reply:
Yes, that is correct. I shall add a comment here.

In order to leverage the NEON functions implemented in [vec256_float_neon.h](https://github.com/pytorch/pytorch/blob/main/aten/src/ATen/cpu/vec/vec256/vec256_float_neon.h), the width needs to be 256.